projects/packages/search/src/dashboard/components/pages/dashboard-page.jsx

```diff
@@ -144,6 +156,35 @@ export default function DashboardPage( { isLoading = false } ) {
 	);
 }
 
+const PlanInfo = props => {
+	const hasIndex = props.hasIndex;
+	const info = props.recordMeterInfo;
+	// Site Info
+	// TODO: Investigate why this isn't returning anything useful.
+	const siteTitle = useSelect( select => select( STORE_ID ).getSiteTitle() ) || 'your site';
+	// Plan Info data
+	const currentPlan = useSelect( select => select( STORE_ID ).getCurrentPlan() );
+	const currentUsage = useSelect( select => select( STORE_ID ).getCurrentUsage() );
+	const latestMonthRequests = useSelect( select => select( STORE_ID ).getLatestMonthRequests() );
+	const tierSlug = useSelect( select => select( STORE_ID ).getTierSlug() );
+	const planInfo = { currentPlan, currentUsage, latestMonthRequests, tierSlug };
```

Comment:
Note that `{} !== {}`, so any component using this variable will re-render even if the object's contents have not changed. I recommend avoiding regrouping property values into new objects in the render function.